django/river/common/database_connection/db_connection.py

```diff
@@ -38,6 +38,7 @@ def build_db_url(db_config):
         host = db_config["host"]
         port = db_config["port"]
         database = db_config["database"]
+        service_name = db_config.get("service")
```

Comment:
Maybe `database = db_config.get("database")` if we need to specify whether `database` or `service`?

Reply:
so we could use a prefix? `database:${database_name}` and `service:${service_name}` ?